jina/peapods/pods/__init__.py

```diff
@@ -569,3 +570,26 @@ def _parse_base_pod_args(self, args):
 
         # note that peas_args['peas'][0] exist either way and carries the original property
         return parsed_args
+
+    def dump(self, pod_name, dump_path, shards, timeout):
+        """Emit a Dump request to its Peas
+
+        :param pod_name: the pod to target
+        :param shards: the nr of shards in the dump
+        :param dump_path: the path to which to dump
+        :param timeout: time to wait (seconds)
+        """
+        for pea in self.peas:
+            if 'head' not in pea.name and 'tail' not in pea.name:
```

Comment:
I think there may be a better way to access peas without head and tail, or be based on some other logic than the name

Reply:
Any suggestions?

Reply:
have a private property for that?